config/settings.yml

```diff
@@ -62,6 +62,7 @@ analytics_debug: true
 min_files_to_search_default: 10
 enabled_features:
   transcripts: false
+  legacy_pdf_viewer: false
```

Comment:
So, by default we're going to use the new implementation in all environments?  The feature here is to go back to the old one?

Reply:
That's not quite what I meant. I meant to move to the browser-based on stage and UAT but not yet to prod.

Reply:
Ahh, ok. So I guess (1) keep this PR but make Legacy the default for now via the feature flag, (2) update this PR to keep the current component names the same and create a "NewPdfComponent" that can be turned on with the feature flag (similar to media).  Net effect is the same, just so naming semantics that come down to the likelihood that we will switch to the new method sooner rather than later.